cpp/inc/bond/protocol/detail/rapidjson_helper.h

```diff
@@ -13,9 +13,34 @@
 #include <boost/call_traits.hpp>
 #include <boost/noncopyable.hpp>
 #include <boost/locale.hpp>
+
 #include "rapidjson/rapidjson.h"
 #include "rapidjson/error/en.h"
+
+// rapidjspn/document.h v1.1 uses std::min/max in ways that conflict
```

Comment:
This mimics logic that has been placed in  rapidjson/document.h after 1.1: https://github.com/miloyip/rapidjson/blob/master/include/rapidjson/document.h#L29

Note that I think there is actually something missing in the fix in rapidjson and that they could probably address the issue more safely by using different syntax around the use of std::numeric_limits<>::max and min.